lib/pages/chat/chat.dart

```diff
@@ -1588,7 +1588,7 @@ class ChatController extends State<Chat>
         await room?.join();
         await waitForRoom;
       },
-    );
+    ).then((_) => _tryLoadTimeline());
```

Comment:
Please show the demo for both case, error and success.

Reply:
IMO: Currently, for the error case, still using old UI from `showFutureLoadingDialog` so If we need to update UI for this I'll create a new ticket and discuss with Design team to improve it.